specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2019-01-01/costmanagement.json

```diff
@@ -831,9 +831,11 @@
           "enum": [
             "WeekToDate",
             "MonthToDate",
+            "BillingMonthToDate",
```

Comment:
@bgsky could you add new enum items to the end?

Reply:
There are some model validation errors. Could you fix them?

Reply:
@sergey-shandar We addressed your comments by moving enum values to the end. In terms of model validation errors. A separate PR fixed them recently, and there is no error from new iteration. Please review and we expect to merge fast.